pages/common/base64.md

```diff
@@ -3,11 +3,11 @@
 > Encode or decode file or standard input to/from Base64, to standard output.
 > More information: <https://www.gnu.org/software/coreutils/manual/html_node/base64-invocation.html>.
 
-- Encode a file:
+- Print a plaintext file's contents encoded in base64:
```

Comment:
```suggestion
- Encode the contents of a file as base64 to stdout:
```

Reply:
Words are hard, thanks for the suggestion!